packages/backend/functions/src/data/repositories/card.repository.js

```diff
@@ -26,7 +26,21 @@ const findAll = async (page = 1, pageSize = 25, sort = {}) => {
   };
 };
 
+const findWithFilters = async (filters) => {
+  const { cardName, supertype, subtype, colors, colorIdentity } = filters
+  return await CardModel.find({
+    $or: [
```

Comment:
Aqui é onde eu to beeemm confuso. Com certeza nao deveria estar usando o $or mas nao to conseguindo entender como fazer pelo mongoose o filter funcionar da maneira correta.

Reply:
Por isso que, se tu nao quer esquentar cabeca com coisas de backend. so faz o endpoint retornar dados mockados :)

Reply:
Mudei pra retornar dados mockados.